Zilon.Core/CDT.LAST.MonoGameClient/Screens/LeaderBoardScreen.cs

```diff
@@ -0,0 +1,85 @@
+namespace CDT.LAST.MonoGameClient.Screens
+{
+    using System;
+
+    using Engine;
+
+    using Resources;
+
+    using Microsoft.Xna.Framework;
+    using Microsoft.Xna.Framework.Graphics;
+    using Microsoft.Xna.Framework.Input;
+
+    /// <summary>
+    /// The leaderboard screen displays all the results of all played users.
+    /// </summary>
+    public class LeaderBoardScreen : GameSceneBase
+    {
+        private readonly TextButton _goToMainMenu;
+
+        private readonly SpriteBatch _spriteBatch;
+
+        /// <inheritdoc />
+        public LeaderBoardScreen(Game game, SpriteBatch spriteBatch)
+            : base(game)
+        {
+            _spriteBatch = spriteBatch;
+
+            var buttonTexture = game.Content.Load<Texture2D>("Sprites/ui/button");
+            var font = Game.Content.Load<SpriteFont>("Fonts/Main");
```

Comment:
You should get main font and button texture from `IUiContentStorage` to prevent multiple loading of same resources.